### ISSUE TITLE
release-please doesn't generate tag or release with Release-As instruction for alpha pre-release

### ISSUE THREAD

**Issue by hbollon** [googleapis/release-please#996]
Hi! 
We are using release-please with Github Action to automate releases on our [Terraboard](https://github.com/camptocamp/terraboard) repository, but we are facing an issue.

We needed to release an alpha version (2.0.0-alpha), to do this we made an empty commit with, as specified in the Readme, a `Release-As: 2.0.0-alpha` instruction.
Once the generated PR was merge, release-please did not generate any tag or release...

I reproduced this in this repo: https://github.com/hbollon/release-please-go-test
I have observed that with our configuration the tag is performed very well without "Release-As" but not with.

Does this problem come from our way of doing things?

#### Environment details

  - OS: Github Action running on ubuntu-latest
  - `release-please` version: latest

#### Steps to reproduce

  1. Do some git stuff triggering release-please PR creation
  2. Do an empty commit with `Release-As: 2.0.0-alpha` message
  3. Squash and merge release-please's PR
  4. No tag or release should be generated

**Comment by bcoe** [comment]
@hbollon I am unable to reproduce your issue, I tested [here](https://github.com/bcoe/test-release-please/pull/76), using `Release-As` to create a pre-release.

One thing I notice is that your [merged PR](https://github.com/hbollon/release-please-go-test/pull/3) is missing the label `autorelease: pending`.... I think what might actually happened is that you were bitten by [this bug](https://github.com/googleapis/release-please/issues/988), which results in PRs being opened repeatedly if you use a pre-release version #.

There's a fix for this incoming [https://github.com/googleapis/release-please/pull/1000].


Let's discuss this problem in #988 👍 I'll make sure `go` is included in the fix.

**Comment by hbollon** [comment]
Hey @bcoe and thanks for your reply!
Indeed, I was aware of this issue and even observed it in our project and the test one but I did not pay attention that there was a difference between the duplicated PRs, I started from the principle that either would have the same effect. Anyways that seems to make sense and I will redo the tests on our side.
In any case, thank you, I will carefully follow this issue and the fix that arrives :smile:
